multimedia/v4l2rtspserver/patches/001_live555_sprintf_format

```diff
@@ -0,0 +1,11 @@
+--- a/live/liveMedia/ServerMediaSession.cpp
++++ b/live/liveMedia/ServerMediaSession.cpp
+@@ -271,7 +271,7 @@ char* ServerMediaSession::generateSDPDes
+ 
+     char const* const sdpPrefixFmt =
+       "v=0\r\n"
+-      "o=- %ld%06ld %d IN %s %s\r\n"
```

Comment:
Is this a time_t thing?

Reply:
yes it is, passing a time_t as %ld produce a SEGV.

Reply:
Yes. Also casting those variables to long long would be a good idea.

Reply:
you are right, it is safer.